QuantConnect.IBAutomater/IBAutomater.cs

```diff
@@ -175,6 +213,11 @@ public void Start(bool waitForExit)
         /// <remarks>The IB Gateway application will be terminated</remarks>
         public void Stop()
         {
+            if (!IsRunning())
```

Comment:
Wondering if `Start/Stop/IsRunning` should share a lock to avoid thread race issues, since their state/result depends on each other